code/modules/admin/verbs/adminjump.dm

```diff
@@ -30,9 +63,9 @@
 	log_admin("[key_name(usr)] jumped to [A]")
 	if(!isobserver(usr))
 		message_admins("[key_name_admin(usr)] jumped to [A]")
-	SSblackbox.record_feedback("tally", "admin_verb", 1, "Jump To Area") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
```

Comment:
Leave these notes in. Not only is it an un-needed diff to remove them, its very important that this is known for anyone who adds blackbox feedback for verbs

Reply:
Not sure why I even removed those to be honest. Presumably because the second parameter is the same across all of them, so I thought the comment was outdated or something.
🤷